example/02/main.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"math/rand"
+
+	"github.com/axamon/gobrain"
```

Comment:
Shouldn't it be `github.com/goml/gobrain`?

Reply:
yea, fixed it